Makefile

```diff
@@ -252,8 +252,6 @@ endif
 bootstrap-only: lerna-bootstrap
 
 yarn-install: clean-all
-	# Install dependencies in individual packages so that we can link them at the top level.
-	for package in eslint/*/; do yarn --ignore-engines --cwd $$package; done
```

Comment:
Don't we still need this until Babel 8?

Reply:
Running yarn install in sub folders incurs changes to the yarn lock files after lerna version, lerna will abort publishing with uncommitted changes. See also the PR descriptions.